### ISSUE TITLE
Naming/UncommunicativeMethodParamName counts double asterisks as offense

### ISSUE THREAD

**Issue by kmoschcau** [bbatsov/rubocop#5627]
The new Naming/UncommunicativeMethodParamName counts double asterisks in method parameter names as an offense.

--------

## Expected behavior

If I just specify `**` as a method parameter in a method with keyword arguments, I expect it to be treated the same way `*` is treated with normal parameters.

## Actual behavior

If the cop encounters `**` it counts it as an offense.

## Steps to reproduce the problem

Write the following code and run rubocop over it.
```ruby
def foo(bar: nil, **)
  puts bar
end
```

## RuboCop version

```
$ rubocop -V
0.53.0 (using Parser 2.5.0.2, running on ruby 2.3.5 x86_64-linux)
```

**Comment by kmoschcau** [comment]
As a further remark, adding `'**'` to the list of `AllowedNames` for the cop does not work either.

**Comment by mikegee** [comment]
I confirmed the issue.

We pull the names of the parameters like so:

https://github.com/bbatsov/rubocop/blob/b65c4d51fea57e458b85d26364da6a4fc0ef2eea/lib/rubocop/cop/mixin/uncommunicative_name.rb#L16

but, the `**` has no children in the AST. It's just a `s(:kwrestarg)`.

Maybe skip arguments without children in the AST?

@kmoschcau here's a workaround until this is addressed:

```yaml
Naming/UncommunicativeMethodParamName:
  AllowedNames:
    - ''
```

**Comment by kmoschcau** [comment]
Thanks for the workaround, that works for now.

**Comment by Drenmi** [comment]
@mikegee: Can not reproduce the issue. Can you confirm it's still there?

**Comment by kmoschcau** [comment]
@Drenmi I just checked again myself and realized I actually already remove the workaround @mikegee posted from my project. I think I remember seeing a remark in the changelog and already seeing it working. I'm going to close this issue, since it's fixed now.
